src/ServiceManagement/Compute/Commands.ServiceManagement.Test/Resources/ReservedIPConfig.cscfg

```diff
@@ -18,7 +18,7 @@
   <NetworkConfiguration>    
     <AddressAssignments>      
       <ReservedIPs>
-        <ReservedIP name="ResrvdIP2"/>
+        <ReservedIP name="{0}"/>
```

Comment:
The current cscfg has a hard-coded reserved ip name.  I want to change this to randomly generated reserved ip name.  This cscfg is now serving as a template and during the test, randomly generated reserved ip name replaces this value.